app/controllers/request_controller.rb

```diff
@@ -1085,4 +1086,20 @@ def redirect_numeric_id_to_url_title
       end
     end
   end
+
+  def redirect_pro_users
+    # Pro users should see their own requests in the pro page, so that if
+    # other site functions send them to a request page, they end up back in
+    # the pro area
+    if feature_enabled?(:alaveteli_pro) && \
```

Comment:
Hmm, interesting. I'd been thinking that since it was your request, you'd want to see it in the same place/livery all the time. However, you're probably right. I guess we need to do a little bit of redirecting both ways, so that an embargoed request goes to the pro page, and then a public one goes to the public version, no matter which url you hit them at.